.github/workflows/Android-CI-release.yml

```diff
@@ -0,0 +1,39 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    name: Publish release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+        with:
+          fetch-depth: 0
+      - name: Install JDK ${{ matrix.java_version }}
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Get the version
+        id: tagger
+        uses: jimschubert/query-tag-action@v2
+        with:
+          skip-unshallow: 'true'
+          abbrev: false
+          commit-ish: HEAD
+      - name: Install Android SDK
+        uses: malinskiy/action-android/install-sdk@release/0.1.1
+      - name: Build project
+        run: ./gradlew clean build
+        env:
+          VERSION: ${{ github.ref }}
+      - run: |
+          assetsAAR=$(find . -name *release.aar | while read -r asset ; do echo "-a $asset" ; done)
+          hub release create ${assetsAAR} -m ${{steps.tagger.outputs.tag}} ${{steps.tagger.outputs.tag}}
```

Comment:
this will attach the artifact to the release?

Reply:
yes